core/_variables.scss

```diff
@@ -1,3 +1,14 @@
+// Breakpoints
+$medium-screen: 600px;
+$large-screen: 900px;
+
+
+// Grid Settings
+$column: 90px;
+$gutter: 30px;
+$grid-columns: 12;
+$max-width: 1200px;
```

Comment:
I think we should remove this block (Lines 6–10):

1. They were commented out before, why uncomment them now?
2. Neat 2.0 will be completely different.